src/plugins/condition/StyleRuleManager.js

```diff
@@ -78,11 +78,13 @@ export default class StyleRuleManager extends EventEmitter {
         this.openmct.objects.get(this.conditionSetIdentifier).then((conditionSetDomainObject) => {
             this.openmct.telemetry.request(conditionSetDomainObject)
                 .then(output => {
-                    if (output && output.length) {
+                    if (output && output.length && (this.conditionSetIdentifier && this.openmct.objects.areIdsEqual(conditionSetDomainObject.identifier, this.conditionSetIdentifier))) {
```

Comment:
I think can add following before , `this.openmct.objects.get( .......`
```
if (!this.conditionSetIdentifier) {
      return;
}
```

Reply:
Ah unfortunatley, this is happening if this.conditionSetIdentifier gets unset AFTER the objects.get happens. That's why I added this check. :/